dockstore-client/src/main/java/io/dockstore/client/cli/nested/WorkflowClient.java

```diff
@@ -411,6 +411,7 @@ public void launch(final List<String> args) {
         if (this.commandLaunch.help) {
             JCommanderUtility.printJCommanderHelpLaunch(jCommander, "dockstore workflow", commandName);
         } else {
+            this.checkIfDockerRunning(); // print a warning message if Docker is not running
```

Comment:
You don't need the `this`, although it doesn't hurt.

Reply:
removed `this`